src/audio/external.c

```diff
@@ -5143,52 +5143,35 @@ void func_800CAC08(void) {
 GLOBAL_ASM("asm/non_matchings/audio/external/func_800CAC08.s")
 #endif
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-void func_800C36C4(?, ?, ?, ?); // extern
-void play_sound(?, ? *, ?, ? *, ? *, ? *); // extern
-extern u8 D_800EA0F0;
-extern u8 D_800EA108;
-extern ? D_800EA10C;
+extern u8 D_800EA10C[];
 extern s8 D_800EA168;
-extern ? D_800EA1C8;
-extern ? D_800EA1D4;
-extern ? D_800EA1DC;
 
-void func_800CAC60(s32 arg0) {
+void func_800CAC60(UNUSED s32 arg0) {
     if ((D_800EA108 == 0) && (D_800EA0F0 == 0)) {
-        play_sound(SOUND_ACTION_EXPLOSION_2, &D_800EA1C8, 0, &D_800EA1D4, &D_800EA1D4, &D_800EA1DC);
-        if ((D_800EA10C.unk0 != 1) && (D_800EA10C.unk1 != 1)) {
-            func_800C36C4(0, 1, 0x37, 5);
-            play_sound(SOUND_ITEM_THUNDERBOLT, &D_800EA1C8, 0, &D_800EA1D4, &D_800EA1D4, &D_800EA1DC);
+        play_sound(0x1900F00CU, D_800EA1C8, 0U, D_800EA1D4, D_800EA1D4, &D_800EA1DC);
+        if ((D_800EA10C[0] != 1) && (D_800EA10C[1] != 1)) {
+            func_800C36C4(0, 1, 0x37U, 5);
+            play_sound(0x5101C00CU, D_800EA1C8, 0U, D_800EA1D4, D_800EA1D4, &D_800EA1DC);
```

Comment:
Any reason you removed the defines? Are you able to put them back in?

Reply:
i can, i just thought i'd clear the entire block inside if the ifndef to keep it clean

Reply:
Please add them back in. These are 100% real.

Reply:
what about the header file?

Reply:
I mean: `SOUND_ACTION_EXPLOSION_2`, `SOUND_ITEM_THUNDERBOLT` these defines.

What about the header file?

Reply:
0x1900F00CU should become SOUND_ACTION_EXPLOSION_2

and 0x5101C00CU SOUND_ITEM_THUNDERBOLT

Reply:
> I mean: `SOUND_ACTION_EXPLOSION_2`, `SOUND_ITEM_THUNDERBOLT` these defines.
> 
> What about the header file?

That the header file already contained the definitions for the defines :) but if there are any like those you mentioned like SOUND_ACTION_EXPLOSION_2, lmk :)

Reply:
![image](https://github.com/n64decomp/mk64/assets/7255464/354f72b6-0a11-44c8-abe3-c6f68b182830)
Please replace `0x1900F00CU` with `SOUND_ACTION_EXPLOSION_2` and `0x5101C00CU` with `SOUND_ITEM_THUNDERBOLT`

Then I'll merge.

Someone documented these hex values and renamed them all to a define for a reason. They need to be left in or we're going to have to re-do work that's already been done.